cmd/kraft/pkg/remove/remove.go

```diff
@@ -0,0 +1,89 @@
+package remove
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/packmanager"
+	"kraftkit.sh/unikraft/app"
+)
+
+type Remove struct {
+	Workdir   string `long:"workdir" short:"w" usage:"workdir location to remove pkg from that location"`
+	Kraftfile string `long:"kraftfile" usage:"Set an alternative path of the Kraftfile"`
+}
+
+func New() *cobra.Command {
+	cmd, err := cmdfactory.New(&Remove{}, cobra.Command{
+		Short: "Remove unikraft library from the project directory",
+		Use:   "remove [FLAGS] [PACKAGE|DIR]",
+		Annotations: map[string]string{
+			cmdfactory.AnnotationHelpGroup: "pkg",
+		},
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	return cmd
+}
+
+func (opts *Remove) Pre(cmd *cobra.Command, _ []string) error {
+	ctx := cmd.Context()
+	pm, err := packmanager.NewUmbrellaManager(ctx)
+	if err != nil {
+		return err
+	}
+
+	cmd.SetContext(packmanager.WithPackageManager(ctx, pm))
+
+	return nil
+}
+
+func (opts *Remove) Run(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return fmt.Errorf("package Name is not specified to remove from the project.")
+	}
+	var workdir string
+	var err error
+
+	if len(args) > 1 {
+		workdir = args[1]
+	} else {
+		workdir = opts.Workdir
+	}
+
+	if workdir == "." || workdir == "" {
+		workdir, err = os.Getwd()
+		workdir += "/"
```

Comment:
You can use functions from the `filepath` package to work with directory paths

Reply:
Done 👍🏽

Reply:
are you sure? the code seems unchanged 😅